posthog/models/cohort/util.py

```diff
@@ -449,28 +449,33 @@ def get_dependent_cohorts(
     seen_cohort_ids = set()
     seen_cohort_ids.add(cohort.id)
 
-    queue = [prop.value for prop in cohort.properties.flat if prop.type == "cohort"]
+    queue = [
+        int(prop.value) for prop in cohort.properties.flat if prop.type == "cohort" and not isinstance(prop.value, list)
```

Comment:
I'm not 100% sure yet, but worth checking if we can ever get here with an invalid cohort id?

I _think_ originally with it being a string, invalid cohort ids (like 'bazinga') are just glossed over, but `int('bazinga')` would raise an error, so worth confirming if there's enough validation before hand that we don't call this function anywhere for invalid cohorts, _or_, we handle them properly.

I do quite like being defensive and ignoring invalid values rather than raising here, so future callers of `get_dependent_cohorts()` don't have to worry about validating things themselves 🤔

Reply:
Good point. I added two try/except blocks to check for this (+ test)

Curious if there's a more idiomatic way of doing this!